src/pages/storage/StoragePoolOverview.tsx

```diff
@@ -38,7 +45,10 @@ const StoragePoolOverview: FC<Props> = ({ pool, project }) => {
               </tr>
               <tr>
                 <th className="u-text--muted">Size</th>
-                <td>
+                <td className="storage-pool-overview-meter">
```

Comment:
```suggestion
                <td className="cluster-member-with-meters">
```
The styling of this one needs adjustments. The height of the member names should be aligned with the meters and there needs to be some gap between the chips and the meters.

![image](https://github.com/user-attachments/assets/5264ef0e-10ab-44da-a114-fe1a05a84404)